.travis.yml

```diff
@@ -37,19 +42,22 @@ before_install:
   - sudo mkdir $HOME
   - sudo chown "$USER:" $HOME
   - LINUXBREW=$HOME/.linuxbrew
-  - git clone https://github.com/Linuxbrew/brew $LINUXBREW
+  - git clone --depth=1 https://github.com/Linuxbrew/brew "$LINUXBREW"
```

Comment:
Let's remove `.git` from Homebrew/science.
*Important note*. `--depth=1` makes the output of `brew --version` less information, and affects info stored in the bottle. So it's fine for `.travis.yml` but no good for `circle.yml` where we build bottles.